src/models/Value.js

```diff
@@ -23,6 +23,7 @@ import { BigNumber } from 'bignumber.js';
 export type TokenValue = {|
```

Comment:
This type got a bit confusing in the multichain world.

Probably should have only following fields: `chain, address, value`, as a minimal asset balance representation. Without `chain` it feels incomplete. Wdyt?

Reply:
I don't think it needs `chain` just because the value is within history that comes for a selected chain and at higher level we already know which chain it is. Same would happen when it comes to sending if we would reuse this type.

Mixed feeling about `symbol` tho. It surely helps when it comes to rendering as we no longer need to lookup for asset just for symbol, but it's not attached to `TokenValue` besides that.

I would keep as is without removing `symbol` as removing symbol would require us to add asset lookups in quite few components.

wdyt?

Reply:
This was intended as general purpose token value structure (not only related to history functionality). So it contained minimal data that identified token type and amount.

IMO I would drop `symbol` as this can be derrived from `address`, but would consider adding `chain` (even if it's duplicating `chain` in history events), because address without chain is not clearly describing asset anymore in multichain app.

Reply:
Nothing really changed since it was the same before with `symbol` since `symbol` can exist in any chain and without `chain` it's the same as it is without `address`. Not sure about applying this fix into this PR 🤔

Reply:
I get your thinking, but if we go this direction it's not just `TokenValue` type and related components.

Reply:
Yeah, this seems more to be technical debt related to mutlichain than something about this PR. maybe just add a TODO comment about debt here.